src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -82,10 +94,10 @@ class SidebarScreen extends Component {
      * Pressing the floating action button to open the CreateMenu modal
      * Selecting an item on CreateMenu or closing it by clicking outside of the modal component
      */
-    toggleCreateMenu() {
-        this.setState(state => ({
-            isCreateMenuActive: !state.isCreateMenuActive,
-        }));
+    hideCreateMenu() {
```

Comment:
> Method called when we click the floating action button
     * will trigger the animation
   
The comment for this function is outdated.

Reply:
Resolved